### PR TITLE
Revise ChatOps not to add go.mod & go.sum when /format runs

### PR DESCRIPTION
**Pull request by rinx**
Signed-off-by: Rintaro Okamura <rintaro.okamura@gmail.com>

### Description:

SSIA

### Environment:

- Golang Version: 1.14.3
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.11.5

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [X] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [X] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [X] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .github/workflows/chatops.yml [hunk 1]
```diff
@@ -448,8 +448,8 @@ jobs:
             git config --global user.name "vdaas-ci"
             git config --global user.email "ci@vdaas.org"
 
-            git add .
-            git commit --signoff -m ":robot: Update license headers and formatting go codes"
+            git add cmd example hack internal k8s pkg
```
Comment (kevindiu) [inline .github/workflows/chatops.yml:451]: how about this?
```suggestion
            git add .
            git rm go.mod go.sum
```

Comment (rinx) [reply .github/workflows/chatops.yml:451]: Thanks.
`git rm go.mod go.sum` removes **go.mod and go.sum file**.
So it should be
```suggestion
            git add .
            git reset go.mod go.sum
```

Comment (rinx) [reply .github/workflows/chatops.yml:451]: how about it?

Comment (kevindiu) [reply .github/workflows/chatops.yml:451]: right... I think you are right :)
LGTM

File: .github/workflows/chatops.yml [hunk 2]
```diff
@@ -435,7 +435,7 @@ jobs:
           sudo make prettier/install
           # sudo make dockfmt/install
 
-          echo "Update license headers and format go codes/yaml/Dockerfile"
+          echo "Update license headers and format go codes/yaml"
 
           make update/goimports
```
Comment (kevindiu) [inline .github/workflows/chatops.yml:440]: BTW start thinking do we need this update goimport?
as user should do the verification before creating PR.

**Pull request merged**